### PR TITLE
DLPJTS-74 use talkeetna-lm for license-managed versions

### PR DESCRIPTION
**Pull request by datalogics-kam**
Use `talkeetna-lm` when applicable for license-managed distributions.
## Testing suggestions
- Use the samples (directory in Talkeetna!) to test; you'll need to put the license file in the samples directory when you do this.
- The license-managed build can be triggered by:
  - `-Dpdfjt-lm`
  - `'-Ppdfjt-lm,!pdfjt-non-lm'`
  - Create a file called `.use-pdfjt-lm`

[DLPJTS-74](https://jira.datalogics.com/browse/DLPJTS-74)

### CODE REVIEW

**Comment by datalogics-devauto** [comment]
Build succeeded.

**Comment by datalogics-devauto** [comment]
Build succeeded.

File: pom.xml [hunk 1]
```diff
@@ -344,12 +356,15 @@
         <property>
           <name>!pdfjt-lm</name>
         </property>
+        <file>
+          <missing>${basedir}/.use-pdfjt-lm</missing>
+        </file>
       </activation>
       <dependencies>
         <dependency>
-          <groupId>com.adobe.pdfjt</groupId>
```
Comment (jrowden8) [inline pom.xml:350]: Nice removal of redundant dependency here because PDFJT is a dependency of Talkeetna.

**Comment by jrowden8** [comment]
This looks good to me! Builds on my machine and I can see that in the dependency tree that the profile "pdfjt-lm" triggers a LM samples build:
com.datalogics.pdf:talkeetna-lm:jar:5.0.0-SNAPSHOT:compile
com.adobe.pdfjt:pdfjt-lm:jar:3.0.1-SNAPSHOT:compile (version selected from constraint [3.0.0,4.0.0))

Removal of the "pdfjt-lm" profile or using "pdfjt-non-lm" profile triggers a non-LM samples build:
com.datalogics.pdf:talkeetna:jar:5.0.0-SNAPSHOT:compile
com.adobe.pdfjt:pdfjt:jar:3.0.1-SNAPSHOT:compile (version selected from constraint [3.0.0,4.0.0))

Great work @datalogics-kam !

**Comment by datalogics-kam** [comment]
Thanks!

**Pull request merged**